js/hoot/control/utilities/Folder.js

```diff
@@ -139,6 +139,32 @@ Hoot.control.utilities.folder = function(context) {
             });
         }
 
+        function updateLastAccessed(node) {
+            var row = d3.select(node.parentNode);
+            row.classed('expiring', true)
+                .append('g').append('svg:foreignObject')
+                .attr('class', 'expiring')
+                .attr('transform', function(d) {
+                    var dd = d.depth-1;
+                    var dy=772+(11*dd);
+                    return 'translate('+ dy +',-9)'; 
+                });
+            var tooltip = d3.select('#datasettable')
+                .append('div')
+                .attr('class', 'tooltip-old-dataset')
+                .text('This dataset has not been used in a while.');
+            d3.selectAll('g.node.expiring')
```

Comment:
could you add this on 145 above right after you `.append('g')`

Reply:
line 145 is appending the alarm clock icon. The tooltip behavior is getting added to the row classed `g.node.expiring`